src/hooks/usePathRewrite.ts

```diff
@@ -58,7 +58,7 @@ export const useQueryRewrite = (): boolean => {
     const [, pathSafe] = currentPath.match(re) || []
 
     if (pathSafe) {
-      const newPath = currentPath.replace(re, '')
+      const newPath = currentPath.replace(pathSafe, '')
```

Comment:
This is the actual fix. It should replace just the address part, not the whole regex. This is how it had been before I broke it in #570.